core/src/main/scala/io/finch/endpoint/param.scala

```diff
@@ -3,31 +3,30 @@ package io.finch.endpoint
 import cats.Id
 import cats.data.NonEmptyList
 import cats.effect.Effect
-import com.twitter.util.{Return, Throw, Try}
 import io.finch._
 import scala.reflect.ClassTag
 
 private[finch] abstract class Param[F[_], G[_], A](name: String)(implicit
   d: DecodeEntity[A],
   tag: ClassTag[A],
   protected val F: Effect[F]
-) extends Endpoint[F, G[A]] with (Try[A] => Try[Output[G[A]]]) { self =>
+) extends Endpoint[F, G[A]] with (Either[Throwable, A] => Either[Throwable, Output[G[A]]]) { self =>
```

Comment:
Same question about `with (Either[Throwable, A] => Either[Throwable, Output[G[A]]])`. It seems to me we can get rid of this now.